providers/flagd/pkg/service/grpc/service.go

```diff
@@ -0,0 +1,179 @@
+package grpc_service
+
+import (
+	ctx "context"
+	"errors"
+
+	flagdModels "github.com/open-feature/flagd/pkg/model"
+	providerModels "github.com/open-feature/golang-sdk-contrib/providers/flagd/pkg/model"
+	of "github.com/open-feature/golang-sdk/pkg/openfeature"
+	log "github.com/sirupsen/logrus"
+	schemaV1 "go.buf.build/grpc/go/open-feature/flagd/schema/v1"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+type GRPCServiceConfiguration struct {
+	Port int32
```

Comment:
same here, i think `uint16` makes more sense. If you agree pls also check other places. :)

Reply:
updated :)